contrib/win32/openssh/OpenSSHBuildHelper.psm1

```diff
@@ -236,17 +237,24 @@ function Start-OpenSSHBootstrap
         }
     }
 
-    #use vs2017 build tool if exists
-    if($VS2017Path -ne $null)
-    {
+    $sdkPath = "${env:ProgramFiles(x86)}\Windows Kits\8.1\bin\x86\register_app.vbs"
 
+    if ($VS2019Path -or $VS2017Path)
+    {
+        # Use VS2019 or VS2017 build tools if installed.
+        if (-not (Test-Path $sdkPath))
+        {
+            $packageName = "windows-sdk-8.1"
```

Comment:
@tgauth, can you please cross check if windows SDK 8.1 is required?
With your recent telemetry changes, we are using windows SDK 10?

Reply:
Yes, the telemetry changes do require windows SDK 10, for the tracelogging header files. 
@heaths, is windows SDK 8.1 a requirement for this? 

Also - the initial merge conflict was caused by changes I made to the build script for telemetry, so I can help resolve to make sure all changes get incorporated.

Reply:
No. This was based on the script at the time but this PR has been open and unmerged for a long time. IMO, would've been better to merge it and add the UI later so any changes to other source would've been accounted for.